### PR TITLE
Fix search issues

### PR DESCRIPTION
**Pull request by chenz4027**
Signed-off-by: Maggie Chen <magchen@redhat.com>
https://github.com/stolostron/backlog/issues/24116
https://github.com/stolostron/backlog/issues/24120

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -563,13 +568,8 @@
...
     "creation.app.section.settings": "Settings: Specify application behavior",
     "creation.app.section.techPreview": "Technology Preview",
-    "creation.app.settings.clusterSelector": "Deploy application resources on clusters with all specified labels",
-    "creation.app.local.clusters.only": "{{0}} deploys only to local cluster",
-    "creation.app.clusters.matching": "{{0}} deploys to clusters with these label: {{1}}",
-    "creation.app.clusters.expressions": "{{0}} deploys to clusters with these expressions: {{1}}",
-    "creation.app.clusters.all.online": "{{0}} deploys to all online clusters and local cluster",
-    "creation.app.clusters.only.online": "{{0}} deploys to all online clusters",
     "creation.app.select.no.existing.placement.rule": "The selected namespace does not contain any existing placement configurations",
+    "creation.app.settings.clusterSelector": "Deploy application resources on clusters with all specified labels",
```
Comment (chenz4027) [inline frontend/public/locales/en/translation.json:572]: Not sure why they are changed, but seems like they are just changed in sequence

**Review by fxiang1 — APPROVED** [review]
LGTM! Thanks Maggie!

File: frontend/public/locales/en/translation.json (hunk 1 above)
Comment (fxiang1) [reply frontend/public/locales/en/translation.json:572]: I think John made some changes in that area recently. Maybe the sequence got changed when he delivered the new code.

Comment (chenz4027) [reply frontend/public/locales/en/translation.json:572]: Ah I see, thanks Feng!

**Pull request merged**